ArticleText.java

```diff
@@ -0,0 +1,18 @@
+import java.io.IOException;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+public class ArticleText {
+	public static void main(String[] args) throws IOException {
+		Document doc = Jsoup.connect("https://www.cnn.com/").get();
```

Comment:
Why don't use args as parameter instead of hard coding "https://www.cnn.com/"

Change to like `Document doc = Jsoup.connect(args[0]).get();`
https://stackoverflow.com/questions/890966/what-is-string-args-parameter-in-main-method-java